hangman/game.py

```diff
@@ -1,25 +1,87 @@
 from .exceptions import *
+import random
 
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
-
-def _get_random_word(list_of_words):
-    pass
-
+""" Mask word"""
 
 def _mask_word(word):
-    pass
+  if word =='':
+    raise InvalidWordException
+    
+  else: 
```

Comment:
can remove `else` .. if the program does not raise `InvalidWordException`, it'll execute the rest of the codes ..